superset/assets/javascripts/SqlLab/components/AceEditorWrapper.jsx

```diff
@@ -8,6 +8,22 @@ import { areArraysShallowEqual } from '../../reduxUtils';
 
 const langTools = ace.acequire('ace/ext/language_tools');
 
+const keywords = (
```

Comment:
it gets evaluated on import so it's probably paid for by webpack initially (though I'm not 100% on that), meaning the user's browser doesn't spend those cycles. From a perf standpoint I think it's a tiny bit more workload for webpack